ipc/provider/src/checkpoint.rs

```diff
@@ -27,6 +28,8 @@ pub struct BottomUpCheckpointManager<T> {
     metadata: CheckpointConfig,
     parent_handler: T,
     child_handler: T,
+    /// The number of blocks away from the chain head that is considered final
+    finalization_blocks: ChainEpoch,
```

Comment:
As described above, I don't think we need this parameter (at least for Tendermint). We can leave it here, but let's make the default 0